### PR TITLE
feature(project): Started the implementation of the Cookies Disclaimer

### PR DESCRIPTION
**Pull request by hamadycisse**
Solves #6

### CODE REVIEW

**Comment by AdderlyMedina** [comment]
Screenshot from 2024-04-14 13-38-03
Screenshot from 2024-04-14 13-38-49

**Comment by carylemedeho** [comment]
@AdderlyMedina the unit tests are failing, you should update the failing snapshot. you can do this by pressing **u** after running **npm run test:unit**

**Comment by carylemedeho** [comment]
@AdderlyMedina Also, after the user has accepted the cookies, the modal should be hidden, missing this requirement right now. If the local storage is cleared or the accept cookies key is deleted, the cookie should appear right away.

File: src/lib/frontend/components/CookiesDisclaimerModal.svelte [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  })
+
+  const updateCookies = (accepted: boolean) => {
+    cookieStore.set(accepted)
+    showCookiesDialog = false
+  }
+</script>
+
+<div
+  class="fixed bottom-0 p-1 right-0 bg-green-50 shadow-md cookies--container {showCookiesDialog ? 'block' : 'hidden'}">
```
Comment (hamadycisse) [inline src/lib/frontend/components/CookiesDisclaimerModal.svelte:20]: Why does name contain two dashes? Is this common practice?

Comment (AdderlyMedina) [reply src/lib/frontend/components/CookiesDisclaimerModal.svelte]: No, it is not a normal practice. I will change it

File: src/lib/frontend/components/CookiesDisclaimerModal.svelte [hunk 2]
```diff
@@ -0,0 +1,54 @@
+<script lang="ts">
+  import { cookieStore, getCookieStore } from '$lib/frontend/stores/localStorageStore/CookieStore'
+  import { onMount } from 'svelte'
+
+  let showCookiesDialog: boolean = false
+
+  onMount(() => {
```
Comment (carylemedeho) [inline src/lib/frontend/components/CookiesDisclaimerModal.svelte:7]: i don't think you need the onMount and the showCookiesDialog variable, the cookie store is a svelte store, you can get the value, subscribe to the store and set the value as well 
image

Comment (hamadycisse) [reply src/lib/frontend/components/CookiesDisclaimerModal.svelte:7]: see https://github.com/6C-Solutions-4-Small-Businesses/online-shop-template/issues/17

**Pull request merged**